cfg/telestrat.cfg

```diff
@@ -4,7 +4,7 @@ SYMBOLS {
     __RAMEND__:    type = weak, value = $9800;
 }
 MEMORY {
-    ZP:      file = "", define = yes, start = $00E0,            size = $001A;
+    ZP:      file = "", define = yes, start = $00B0,            size = $003A;
```

Comment:
Why was $B0-$DF not used before? Was that just a bug that is now fixed? Or does using $B0-$DF imply some limitation / compatibility reduction?

Reply:
Hello,

$00 to $8B are used by the main telestrat bank (telemon : main 16 KB bank.)
$8c to $CF are used by Hyperbasic language (another 16KB bank)
$D0 to $FF are used by Telematic (for minitel routines).

The first time, i used telematic cardridge ZP.  Now, i grab also Hyperbasic ZP because this cardridge won't work (i don't have source code yet). For instance, i manage this target with telemon primitives only for instance.

If you want more info, you can check this : http://orix.oric.org/doku.php

or 

https://www.youtube.com/watch?v=VKgHpLKjl9c&t=38s

Reply:
Thanks for the clarification.

Is there a specific reaon why you then don't go  for `start = $008C, size = $0074` ?

Note: The reason why other target tend to have a `size = $001A` is (or should be) that they save and restore the ZP locations used. So they want to minimize the save area.

Reply:
Ooops, I just noticed that you save and restore too. That answers my question ;-)